src/components/QueryBuilder.vue

```diff
@@ -58,9 +63,18 @@ export default Vue.extend( {
 				property: null as null | Error,
 				value: null as null | Error,
 			},
+			optionItems: [
```

Comment:
Those can be inside the ValueTypeDropdown as well. They Idea of this component is to encapsulate all the things that are specific to this particular Dropdown.

Reply:
right. I moved it out of the query builder